File: Golang/1007.go
package main

import (
	"fmt"
)

func main() {
//DIFERENCA = (A * B - C * D).

  var a,b,c,d int

  fmt.Scan(&a)
  fmt.Scan(&b)
  fmt.Scan(&c)
  fmt.Scan(&d)
  
  diferenca := ( a * b - c * d )

  fmt.Printf("DIFERENCA = %d\n", diferenca)
}

File: Elixir/1007.exs
a = IO.gets("") |> String.trim() |> String.to_integer
b = IO.gets("") |> String.trim() |> String.to_integer
c = IO.gets("") |> String.trim() |> String.to_integer
d = IO.gets("") |> String.trim() |> String.to_integer

diferenca = (a * b - c * d)
IO.puts "DIFERENCA = #{diferenca}"
